.github/workflows/php_tests.yml

```diff
@@ -11,21 +11,20 @@ jobs:
         uses: actions/checkout@v2
         with:
           submodules: recursive
-      - name: Switch to PHP 8.1
-        run: |
-          sudo update-alternatives --set php /usr/bin/php8.1
-          sudo update-alternatives --set phar /usr/bin/phar8.1
-          sudo update-alternatives --set phar.phar /usr/bin/phar.phar8.1
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: '7.4'
```

Comment:
We have to use PHP 7.x to be compatible with PHPUnit 7 which we want to use sometimes (more below).